src/scitokens/scitokens.py

```diff
@@ -39,6 +40,7 @@ def __init__(self, key=None, key_id=None, parent=None, claims=None):
             raise NotImplementedError()
 
         self._key = key
+        self._key_alg = algorithm if algorithm is not None else "RS256"
```

Comment:
Can we set the default via the configuration system?

Additionally, does it make any sense to do any sort of input validation here?  Particularly, it doesn't make sense to use the various HMAC modes...